content/en/docs/concepts/services-networking/service.md

```diff
@@ -604,6 +604,10 @@ Specify the assigned IP address as loadBalancerIP. Ensure that you have updated
 
 {{< /note >}}
 
+#### Disabling Node Ports
+
+TODO
+
```

Comment:
We should also document that a LoadBalancer service comes, by default, with a NodePort.

Reply:
Good point, will add some words for that